docs/attributes-registry/container.md

```diff
@@ -22,7 +22,7 @@ A container instance.
 | <a id="container-image-name" href="#container-image-name">`container.image.name`</a> | string | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` | ![Development](https://img.shields.io/badge/-development-blue) |
 | <a id="container-image-repo-digests" href="#container-image-repo-digests">`container.image.repo_digests`</a> | string[] | Repo digests of the container image as provided by the container runtime. [5] | `["example@sha256:afcc7f1ac1b49db317a7196c902e61c6c3c4607d63599ee1a82d702d249a0ccb", "internal.registry.example.com:5000/example@sha256:b69959407d21e8a062e0416bf13405bb2b71ed7a84dde4158ebafacfa06f5578"]` | ![Development](https://img.shields.io/badge/-development-blue) |
 | <a id="container-image-tags" href="#container-image-tags">`container.image.tags`</a> | string[] | Container image tags. An example can be found in [Docker Image Inspect](https://docs.docker.com/engine/api/v1.43/#tag/Image/operation/ImageInspect). Should be only the `<tag>` section of the full name for example from `registry.example.com/my-org/my-image:<tag>`. | `["v1.27.1", "3.5.7-0"]` | ![Development](https://img.shields.io/badge/-development-blue) |
-| <a id="container-label" href="#container-label">`container.label.<key>`</a> | string | Container labels, `<key>` being the label name, the value being the label value. | `container.label.app=nginx` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="container-label" href="#container-label">`container.label.<key>`</a> | template[string] | Container labels, `<key>` being the label name, the value being the label value. | `nginx` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
at first glance, it seems like changing `string` -> `template[string]` goes in the other direction of these "just being strings"?

Reply:
yeah, I think you're right. Do you think `<key>` + some language in the brief is enough to indicate that this attribute is slightly special?

Reply:
rolled this part back